### PR TITLE
Docs: Added documentation for 'value' option in variable syntax

### PR DESCRIPTION
**Pull request by hfp-robloh**
**What is this feature?**

Please see title

**Why do we need this feature?**

The option 'value' was missing from the documentation and is required to extract the value from key-value variables (e.g. myvar = 'key : val').

**Who is this feature for?**

Dashboard users using key-value variables.

**Which issue(s) does this PR fix?**:

Fixes #

Missing documentation

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by imatwawana** [comment]
Hi @hfp-robloh - Thanks for the contribution! Before I tag in the dashboards team to give this technical review, I just want to make sure I'm understanding your update. Can you show me a screenshot of what you're documenting in the UI?

**Comment by hfp-robloh** [comment]
Hi @imatwawana , 

Below is a screenshot from the .md preview on github with my changes identified. Everything one would need to understand why this change is necessary can be found here: https://community.grafana.com/t/change-title-based-on-condition-repeating-panel/118953/12

image

**Comment by diegommm** [comment]
Hi! Greetings from the Release Guild. Just letting you know I'm updating your PR's milestone to [11.2.x](https://github.com/grafana/grafana/milestone/529). If you need to see your changes in a previous version, please consider adding a backport label.

Thank you!

File: docs/sources/dashboards/variables/variable-syntax/index.md [hunk 1]
```diff
@@ -176,14 +176,24 @@ Interpolation result: "'test''1','test2'"
...
 Interpolation result: "test1 + test2"
 ```
 
+### Value (Key-value variables)
+
+Returns the value of a key-value variable.
+
+```bash
+servers = ["test1 : 3"]
+String to interpolate: '${servers:value}'
```
Comment (axelavargas) [inline docs/sources/dashboards/variables/variable-syntax/index.md:193]: 🤔 we don't support this `value` format, are you using a fork of Grafana with this implementation? 

In the screen recording, you will see the console prints an error and then it fallbacks to the global format, which in the case of `key-value` variables returns the value. 

https://github.com/grafana/grafana/assets/239999/8e2c0b75-7034-49aa-a9f5-79a8026cbbd8

**Comment by axelavargas** [comment]
Hey @hfp-robloh , I am closing the PR as we don't support the `value` format. I left a comment during my review  😄.
If you have any comments let us know, 
Thank You 🙏🏾

**Comment by hfp-robloh** [comment]
Hi Alexa V,

There definitely is something to be added to the docs here (They are incomplete when it comes to accessing the key or value of a key-value variable). The problem is summarized here and in the messages that follow this one: https://community.grafana.com/t/change-title-based-on-condition-repeating-panel/118953/13

Luckily I was able to solve my problem, and documented how I achieved it for the next person who might need it (I have already seen several such people). Therefore I think this deserves a closer look. I hope this makes Grafana better!

Kind regards,
Robin L.

Von: Alexa V ***@***.***>
Gesendet: Donnerstag, 11. Juli 2024 09:27
An: grafana/grafana ***@***.***>
Cc: Robin Loh ***@***.***>; Mention ***@***.***>
Betreff: Re: [grafana/grafana] Docs: Added documentation for 'value' option in variable syntax (PR #86317)


Hey @hfp-robloh<https://github.com/hfp-robloh> , I am closing the PR as we don't support the value format. I left a comment during my review 😄.
If you have any comments let us know,
Thank You 🙏🏾

**Pull request closed without merging**